src/osparc_control/errors.py

```diff
@@ -0,0 +1,14 @@
+class BaseControlError(Exception):
+    """inherited by all exceptions in this module"""
+
+
+class NoReplyError(BaseControlError):
+    """Used when retrying for a result"""
+
+
+class CommnadNotAcceptedError(BaseControlError):
+    """Command was not accepted by remote"""
+
+
+class NoCommandReceivedArrivedError(BaseControlError):
```

Comment:
is that not some kind of timeout error?

Reply:
Yes, I've changed the name to make it more obvious